core/src/test/java/edu/wpi/grip/core/sources/NetworkTableEntrySourceTest.java

```diff
@@ -41,16 +43,34 @@ public NetworkTableEntrySourceTest() {
 
   @Before
   public void setUp() {
-    testingNtManager = p -> new MockNTReceiver(p, testTable);
-
-    testTable.putBoolean(BOOLEAN_PATH, true);
-    testTable.putDouble(NUMBER_PATH, TEST_NUMBER);
-    testTable.putString(STRING_PATH, TEST_STRING);
+    ntInstance = NetworkTableInstance.create();
```

Comment:
Hate this dam singleton pattern.

Reply:
It's not a singleton? There's a default instance, but `create()` will return a new instance. Makes it useful for tests since there's no global state

Reply:
Oh. Cool!